pkg/pillar/types/zedroutertypes.go

```diff
@@ -2872,3 +2904,25 @@ func (status OnboardingStatus) LogDelete(logBase *base.LogObject) {
 func (status OnboardingStatus) LogKey() string {
 	return string(base.OnboardingStatusLogType) + "-" + status.Key()
 }
+
+// AppNumber :
+// PS. Any change to AppNumMax, must be
+// reflected in the BitMap Size(8 bytes)
+const (
+	AppNumStart   = 0   // DhcpRange start
+	AppNumEnd     = 253 // DhcpRange end
+	AppNumMax     = 254 // Max Count
+	AppNumBridge  = 255 // For Bridge
```

Comment:
Today we give the bridge the first IP address. Does this use of 255 have any impact on the IP address which will be assigned to the bridge?

Reply:
Have moved the first IP address (default) value to -2, to avoid any collisions inside DhcpRange. The Gateway address can be any host IP address in the subnet range.